app/views/ontologies/browser/browse.html.haml

```diff
@@ -1,16 +1,14 @@
 .browse-center
   .browse-container
-
-    .browse-submit-new-ontology-and-you-are-admin-container
+    .container.align-alert
```

Comment:
the alert can be a little bigger in the width, no?

Reply:
I tried to do it with the same width as the old block =>  to confirm with @jonquet